### PR TITLE
Remove payload2 from value bridging interface

### PR DESCRIPTION
**Pull request by kateinoigakukun**
Depends on #63

### CODE REVIEW

**Review by MaxDesiatov — APPROVED** [review]
This seems legit as the tests pass, but could we have a doc comment somewhere that clarifies the purpose, the number, and the types of payloads? Is was going to provide a write up on JavaScriptKit internals, would be very interested to know more in details, if possible 🙏

File: Sources/JavaScriptKit/JSValueConvertible.swift [hunk 1]
```diff
@@ -194,8 +194,6 @@ extension RawJSValue: JSValueConvertible {
             return .undefined
         case .function:
             return .function(JSFunction(id: UInt32(payload1)))
-        default:
-            fatalError("unreachable")
```
Comment (j-f1) [inline Sources/JavaScriptKit/JSValueConvertible.swift:198]: What happens if this is passed an invalid `kind`?

Comment (kateinoigakukun) [reply Sources/JavaScriptKit/JSValueConvertible.swift:198]: The swift runtime will crash with error message that says the given enum value has invalid raw value.

**Review by j-f1 — APPROVED** [review]

**Comment by MaxDesiatov** [comment]
Thanks a lot for the diagrams! Happy for this to be merged, and maybe if you folks manage to review #62, it would be a good 0.7.0 release? 😉

**Pull request merged**